README.rst

```diff
@@ -4,7 +4,7 @@ SageMaker TensorFlow
 .. role:: python(code)
    :language: python
 
-SageMaker specific extensions to TensorFlow, for Python 2.7, 3.4-3.6 and TensorFlow versions 1.7-1.12. This package includes the :python:`PipeModeDataset` class, that allows SageMaker Pipe Mode channels to be read using TensorFlow Datasets.
+SageMaker specific extensions to TensorFlow, for Python 2.7, 3.4-3.7 and TensorFlow versions 1.7-1.15.2 (Python 3.7 support is only available for TensorFlow 1.15.2), and for Python 3.6-3.7. This package includes the :python:`PipeModeDataset` class, that allows SageMaker Pipe Mode channels to be read using TensorFlow Datasets.
```

Comment:
how about TF 2.0, 2.1?

Reply:
TF 2.0 and 2.1 are in a different branch: https://github.com/aws/sagemaker-tensorflow-extensions/tree/tf-2 but looks like it was never updated to reflect difference between 2 branches. I will submit another PR to the tf-2 branch.